__tests__/app.test.js

```diff
@@ -438,3 +455,29 @@ test("Status 404 path not found, if review_id entered is valid but does not exis
       expect(msg).toBe("Path not found");
     });
 });
+
+test("Delete /api/comments/:comment_id Status 204 by comment_id", () => {
+  return request(app)
+    .delete("/api/comments/2")
+    .expect(204)
+    .then(({ body }) => {
+      expect(body).toEqual({});
+    });
+});
+
+test("status 400 Bad request when comment is not valid data type", () => {
+  return request(app)
```

Comment:
be clearer in your description - it's the comment id

Reply:
Sorted these errors (renamed both tests)